cpp/hide-formfields/CMakeLists.txt

```diff
@@ -1,26 +1,22 @@
 cmake_minimum_required(VERSION 3.10)
 
-set (CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
To create a new PR just for this request ? the space or the whole modifications for CMake ?

If for the whole CMake, do i need to close this one in order to open two different , one for configs and the second for cmake ?

Reply:
Yeah I guess I'd just prefer a clean PR, as these changes to the hide-formfields CMake have nothing to do with the issues you described and fixed. Also, FWIW, I'm not even sure we build the hide-formfields executable for the Docker image anymore, it's been deprecated. Did you have a reason for using it and compiling it?

Reply:
Thank you for your feedback! Just opened two fresh PR for thoses changes.

The reason I made changes to the hide-formfields CMake is because I was trying to run the Node.js server locally (not using Docker). 

To do so, I needed to specify the path to the `hide-formfields` executable in the `.env` file, but the build process was failing with the previous configuration. Updating the CMake configuration was necessary to resolve those issues and get the build working again.

If the `hide-formfields` executable is deprecated and no longer in use, I can remove this part from the PR and focus solely on the changes related to the server configuration. Let me know how you'd prefer to proceed!